src/dom/index.js

```diff
@@ -80,7 +80,11 @@ export function setAccessor(node, name, old, value, isSvg) {
 		(node._listeners || (node._listeners = {}))[name] = value;
 	}
 	else if (name!=='list' && name!=='type' && !isSvg && name in node) {
-		setProperty(node, name, value==null ? '' : value);
+		// Attempt to set a DOM property to the given value.
+		// IE & FF throw for certain property-value combinations.
+		try {
+			node[name] =  value==null ? '' : value;
```

Comment:
Two spaces or a tab after the equals sign :)

Reply:
Whoops - amended my commit with the fix